app/main/activity/crowdsourced-survey-table.directive.js

```diff
@@ -0,0 +1,97 @@
+module.exports = CrowdsourcedSurveyTable;
+
+function CrowdsourcedSurveyTable() {
+    return {
+        restrict: 'E',
+        scope: {
+            filters: '='
+        },
+        controller: CrowdsourcedSurveyTableController,
+        template: require('./crowdsourced-survey-table.html')
+    };
+}
+
+CrowdsourcedSurveyTableController.$inject = ['$scope', '$translate', 'FormEndpoint', 'FormStatsEndpoint', 'FormContactEndpoint', 'PostEndpoint', '_', 'PostFilters', 'Session'];
+function CrowdsourcedSurveyTableController($scope, $translate, FormEndpoint, FormStatsEndpoint, FormContactEndpoint, PostEndpoint, _, PostFilters, Session) {
+    $scope.crowdsourcedSurveyStatsByForm = [];
+    $scope.totalAllReduced = 0;
+    $scope.totalWebReduced = 0;
+    $scope.totalEmailReduced = 0;
```

Comment:
maybe a silly question but... why "reduced"?

Reply:
Because originally I was going to iterate of the forms and reduce the number for each data source. But then I realized that would require an extra loop. I suppose total and reduced is redundant though. I'll change it :)

Reply:
Cool. Ping me when ready :D